__test__/shop.test.js

```diff
@@ -0,0 +1,89 @@
+var Shop = require('../src/shop');
+var Item = require('../src/item');
```

Comment:
It's preferred to use consts or lets in javascript now. See this:
https://github.com/airbnb/javascript#variables--const

Reply:
k